data-raw/northern-ireland-hsct-secondary-care.R

```diff
@@ -0,0 +1,147 @@
+# Northern Ireland - Secondary Care Indicators
+# Health and Social Care Trusts
+
+# ---- Load libs & helpers ----
+library(tidyverse)
+library(geographr)
+library(sf)
+library(healthyr)
+library(demographr)
+library(compositr)
+library(ggridges)
+library(devtools)
+library(readODS)
+
+hsct <- boundaries_trusts_ni18 |>
+  st_drop_geometry()
+
+lookup_trusts <- geographr::boundaries_trusts_ni18 |>
+  st_drop_geometry()
+
+# Population counts
+# Source: https://www.ninis2.nisra.gov.uk/public/AreaProfileReportViewer.aspx?FromAPAddressMulipleRecords=Northern@@Northern@20?
+
+url <- "https://www.ninis2.nisra.gov.uk/Download/Population/Population%20Estimates%20Broad%20Age%20Bands%20(administrative%20geographies).ods"
+
+pop_raw <- download_file(url, ".ods")
+
+pop_ni_hsct <- read_ods(
+  pop_raw,
+  sheet = "HSCT",
+  range = "A4:C9"
+) |>
+  select(
+    trust18_name = "HSCT",
+    trust18_code = "HSCT Code",
+    population = "Persons"
+  )
+
```

Comment:
Should we add this data first to demographr? What do you think?

Reply:
We could do, I was just using the scotland_hb_secondary_care code as a guide. So far, I have not needed to use this elsewhere, but in the future it could be useful to draw from the demographr package. I am open to adding it :)